src/e2e-test/java/io.cdap.plugin/mssql/hooks/MssqlTestSetUpHooks.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.mssql.hooks;
+
+import com.google.cloud.bigquery.BigQueryException;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.mssql.utils.MssqlClient;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+/**
+ * Mssql test hooks.
+ */
+public class MssqlTestSetUpHooks {
+
+  public static String tableName = PluginPropertyUtils.pluginProp("mssqlSourceTable");
+  public static String schemaName = PluginPropertyUtils.pluginProp("mssqlSchema");
+  public static String datatypeColumns = PluginPropertyUtils.pluginProp("mssqlDatatypeColumns");
+
+  @Before(order = 1)
+  public static void setTableName() {
+    String randomString = RandomStringUtils.randomAlphabetic(10);
+    String sourceTableName = String.format("SourceTable_%s", randomString);
+    PluginPropertyUtils.addPluginProp("mssqlSourceTable", sourceTableName);
+  }
+
+  @Before(order = 2, value = "@MSSQL_SOURCE")
+  public static void createTable() throws SQLException, ClassNotFoundException {
+    MssqlClient.createTable(tableName, schemaName, datatypeColumns);
+    BeforeActions.scenario.write("MSSQL Source Table - " + tableName + " created successfully");
+  }
+
+  @After(order = 2, value = "@MSSQL_DELETE")
+  public static void dropTable() throws SQLException, ClassNotFoundException {
+    MssqlClient.deleteTable(schemaName, tableName);
+    BeforeActions.scenario.write("MSSQL Source Table - " + tableName + " deleted successfully");
+  }
+
+  @After(order = 1, value = "@BQ_SINK_TEST")
+  public static void deleteTempTargetBQTable() throws IOException, InterruptedException {
```

Comment:
Update the name of the property and we may need a hook to create a target table as well.